Podfile

```diff
@@ -161,7 +161,7 @@ abstract_target 'Apps' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.55.0'
+    gutenberg :commit => '396d23c243956729cf412fbd47366f7e8e63aad9'
```

Comment:
@fluiddot Do you plan to wait to merge this till `v1.55.1` lands and its tag has been pushed to the GitHub?

Reply:
Yeah, this PR will only be merged after the `v1.55.1` tag is created.